5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.driver.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import TodoListItem from "./TodoListItem";
+import { BaseDriver } from "../../../test/baseDriver";
+import dataHooks from "../../../dataHooks/dataHooks";
+import { mockTodo, throwErrorOnMissingWrapper } from "../../../test/utils";
+import { Item } from "../../../interfaces/interfaces";
+
+export class TodoListItemDriver extends BaseDriver {
+  private todoItem: Item = mockTodo();
+  private removeTodo: jest.Mock = jest.fn();
+
+  private buttonClickHandler = (dataHook: string) => {
+    if (!this.wrapper) return throwErrorOnMissingWrapper();
+    const buttonElem = this.wrapper.getByTestId(dataHook);
+    fireEvent.click(buttonElem);
+    return buttonElem;
+  };
+
+  private getTodoText = () => {
+    return this.wrapper?.getByText(this.todoItem.text).innerHTML;
+  };
+
+  private getTodoStatus = () => {
+    const switchElem = this.wrapper?.getByTestId(
```

Comment:
there's inconsistency here - some times you're using `this.wrapper?` while other times `if (!this.wrapper)..` use one method everywhere (the latter 😄 )

Reply:
I let the autocomplete to control my coding style... I'll change that 😅